flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/rest/header/session/TriggerSessionHeartbeatHeaders.java

```diff
@@ -76,4 +76,9 @@ public String getTargetRestEndpointURL() {
     public static TriggerSessionHeartbeatHeaders getInstance() {
         return INSTANCE;
     }
+
+    @Override
+    public String operationId() {
+        return "TriggerSession";
```

Comment:
Why is it not return `TriggerSessionHeartbeat` here? Is there any other document specifying the name ? I just noticed that other operationId are generated by class name without headers.

Reply:
Done.